scripts/install-aica-docker.sh

```diff
@@ -1,10 +1,50 @@
 #!/bin/bash
 
+function get_shell_rc_path () {
+  local user home_dir
+  user=$(logname)
+  home_dir=$(eval echo "~${user}")
+  if [[ "${SHELL##*|}" == *"zsh"* ]]; then
+    echo "${home_dir}/.zshrc"
+  elif [[ "${SHELL##*|}" == *"bash"* ]]; then
+    if [[ "$OSTYPE" != "darwin"* ]]; then
+      echo "${home_dir}/.bashrc"
+    else
+      echo "${home_dir}/.bashrc"
+    fi
+  else
+    echo "Currently, only zsh and bash shells are supported for autocompletion with aica-docker."
+    exit 1
+  fi
+}
+
+function source_completion_script () {
+  grep -v /src/aica-docker-completion.sh "$1" > tmpfile && mv tmpfile "$1"
+  echo "source ${SCRIPT_DIR}/src/aica-docker-completion.sh" >> "$1"
+}
+
 if [[ "$OSTYPE" != "darwin"* ]]; then
   mkdir -p /usr/local/bin
 fi
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"/aica-docker.sh
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+SYMLINK="/usr/local/bin/aica-docker"
+
+if test -f "$SYMLINK"; then
+  echo "Updating symbolic link from ${SYMLINK} to ${SCRIPT_DIR}/aica-docker.sh;"
+  sudo rm "${SYMLINK}" || exit 1
+else
+  echo "Creating a symbolic link from ${SYMLINK} to ${SCRIPT_DIR}/aica-docker.sh;"
+fi
+sudo ln -s "${SCRIPT_DIR}/aica-docker.sh" ${SYMLINK}
 
-echo "Creating a symbolic link from /usr/local/bin/aica-docker to ${SCRIPT_DIR}"
-ln -s "${SCRIPT_DIR}" /usr/local/bin/aica-docker
+SHELL_RC_PATH=$(get_shell_rc_path)
+while true; do
+  read -r -p "Do you with to install auto completion for aica-docker to '${SHELL_RC_PATH}'? [yes|no]
+[yes] >>>" yn
```

Comment:
This line is a bit weird as `yes` is not a default value so I don't know why it is there. Just put nothing before the read input. I would also suggest to make a default value, either yes or no let's say yes because most probably this is the choice wanted. So basically if I press enter then it is the yes choice, anything else is no.

Reply:
Okay but if `yes` if the default option, i should leave it there no?

Reply:
Usually we put it capital in the `[Y|n]` on top

Reply:
that indicates it is default or you actually write `default = yes`